modules/preview-react/form-field/stories/FormField.stories.mdx

```diff
@@ -22,8 +23,19 @@ yarn add @workday/canvas-kit-preview-react
 
 ### Customizing With Behavior Hooks Example
 
-If you need full customization you can use the form field behavior hooks to build your own solution.
-It is also easy it work with custom components or third party libraries and get the CKR
+If you need full customization you can use the `FormField` behavior hooks to build your own
+solution. It is also easy it work with custom components or third party libraries and get the CKR
```

Comment:
May as well fix this typo where you're at it: "It is also easy it work with custom components..."

Reply:
Thanks!